mailchimp-connector/module.yaml

```diff
@@ -0,0 +1,6 @@
+version: 1.0.0
+dependencies:
+  core:
+    # Required for environmental variables
+    # Ref: https://docs.magnolia-cms.com/product-docs/6.2/Administration/Architecture/Configuration-management.html#_environment_variables
+    version: 6.2.9/*
```

Comment:
There is a dependency on the rest-client module, it should be declared here for completeness sake

Reply:
Added